.github/workflows/avm.template.module.yml

```diff
@@ -99,7 +99,7 @@ jobs:
         uses: ./.github/actions/templates/avm-validateModuleDeployment
         with:
           templateFilePath: "${{ inputs.modulePath }}/${{ matrix.moduleTestFilePaths }}"
-          location: "WestEurope"
+          location: "NorthEurope"
```

Comment:
Was that intended?

Reply:
Only for the time of testing and is now reverted to the initial value. See also the note in the Description/Pipeline reference